oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
> server-side application

It's a good thing this is oauth2*client*, heheh.

(please excuse my bad joke.)